ntbs-service-unit-tests/Jobs/DataQualityAlertsJobTest.cs

```diff
@@ -36,6 +36,11 @@ public async Task OnRun_WithEligibleNotifications_CallsExpectedMethodWithEligibl
             {
                 new Notification {NotificationId = 1}
             });
+            var testDuplicateNotificationIdsList = Task.FromResult(
+                (IList<DataQualityRepository.NotificationAndDuplicateIds>)new List<DataQualityRepository.NotificationAndDuplicateIds>
```

Comment:
It's a little mucky having this class in the DataQualityRepository here but rarely used outside it apart from testing so I'm not too worried